src/smartpeak/include/SmartPeak/core/Parameters.h

```diff
@@ -94,7 +102,17 @@ namespace SmartPeak
 
     Parameter* findParameter(const std::string& parameter);
     const std::string& getFunctionName() const { return function_name_; };
+
+    /**
+    Add a parameter. Doesn't Replace if already exists
```

Comment:
Does it make sense to add in a description of the parameters to the docstrings here and below?